test/react-native/features/fixtures/app/App.js

```diff
@@ -0,0 +1,106 @@
+import React, {Component} from 'react';
```

Comment:
this file mixes single/double quotes and I guess is just generally not linted.

Would be good to extend the eslint configuration to cover these files and then fix the lint errors.